agent/bench-scripts/pbench-trafficgen

```diff
@@ -43,7 +43,7 @@ traffic_generator="trex-txrx" # can be either trex-txrx or moongen-txrx
 trex_use_ht="n"
 trex_use_l2="n"
 benchmark_run_dir=""
-traffic_directions="bidirec"
+traffic_directions="bidirectional"
```

Comment:
Do we need to be binary compatible, and support `bidirec` to mean `bidirectional`?

Reply:
No.  There are a lot more changes coming like this,  we are going to be changing the entire option structure and don't have any need for backwards compatibility.

Reply:
Shall we make these changes in 0.52 instead of 0.51 so that you can accumulate a number of them into one release?

Reply:
The option structure changes are not pending real soon, that is more a forward looking viewpoint.  The changes and additions in this PR are needed to deploy new functionality to users sooner rather than later.